apps/agent/src/login-form/LoginForm.tsx

```diff
@@ -61,6 +65,22 @@ export const LoginForm: React.FC<LoginForm.Props> = (props) => {
         </TextContent>
       </CardHeader>
 
+      <Divider style={{ backgroundColor: "#f5f5f5", height: "10px", border: "none" }} />
+
+      <CardBody id="card-body-note" style={{ backgroundColor: "#ffffff", border: "1px solid #d2d2d2", padding: "1rem" }}>
+        <TextContent style={{ display: "flex", alignItems: "center", gap: "0.5rem" }}>
+          <InfoCircleIcon color="#007bff" />
+          <Text component="p" style={{ color: "#007bff" }}>
+            Access control
+          </Text>
+        </TextContent>
+        <Text component="p" style={{ marginTop: "0.5rem", color: "#6a6e73", marginLeft: "1.5rem" }}>
```

Comment:
<Text component="p" style={{ marginTop: "0.5rem", color: "#6a6e73", marginLeft: "1.5rem" }}>

Reply:
What is the change here?

Reply:
Sorry, you have to remove the color:
` <Text component="p" style={{ marginTop: "0.5rem", marginLeft: "1.5rem" }}>`

Reply:
Done